### ISSUE TITLE
Live Preview: Make the "Demo site" button more prominent

### ISSUE THREAD

**Issue by fushar** [Automattic/wp-calypso#86144]
### Context

With the new Live Preview feature, we now show the "Demo site" button differently depending on the theme.

Before we had this Live Preview feature, here's how a user can find the demo site button (previously called "Live demo"):

|Theme showcase grid|<img width="358" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/6fb5cede-37ce-4095-833a-55c597643bb3">|
|-|-|
|Theme details page|<img width="358" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/c389139f-c390-42d7-902a-5c9e8ad69ff1">|

After we launched the Live Preview feature:

||Live Preview NOT available|Live Preview available|
|-|-|-|
|Theme showcase grid|<img width="354" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/6a1cf805-74c1-4f30-a576-bf63aa4d4cb8">|N/A|
|Theme details page|<img width="1393" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/5cd18e5d-311c-4c16-8a2b-79844884b54e">|<img width="1377" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/75f16cfe-c5a6-4e8b-aa29-a33bfe720159">|

We can see that when a theme has Live Preview available, the "demo site" button is very hidden -- we need to go to the theme details page, and hover on the theme preview on the right.

This is the discussion on the decision to put the the buttons this way: https://github.com/Automattic/dotcom-forge/issues/2391#issuecomment-1599401582.

### Problem

Before we launched the Live Preview feature, it was quite easy to access the "demo site" button from both the theme showcase and theme details page. After we launched the feature, it looks like there have been a quite number of requests to make the buttons more visible again:

> Choosing a theme for a new site has always been my favorite activity. In the past I loved browsing our theme catalog and looking at the demo one by one, checking the typography, how the single post page would look like, and so on.
> 
> This is where I ran into a problem: it’s now difficult for me to find a link to a theme’s demo.

(pekYwv-3Dk-p2)

> Now that we have this feature, when we are in the LiTS there is no way to reach the demo of a theme. It would be great if we could bring it back to quickly grab links to demos or simply so that users can check out the demo before previewing and customizing.

(pekYwv-3ut-p2)

> The "Open live demo" button and the UX to upgrade from the demo site can be crucial for conversions.
> As a user, I can only decide if it's worth it to activate/buy a theme after seeing the live demo to try the theme features.

(p1704770167885199-slack-CRWCHQGUB)

### Ask

Can we request a solution from designers on how to solve this problem?

cc: @Automattic/lego @lucasmendes-design @matt-west

**Comment by arthur791004** [comment]
> It would be great if we could bring it back to quickly grab links to demos or simply so that users can check out the demo before previewing and customizing.

I agree with this. I think we can open the live preview to preview the demo site first and then have a button there to allow people to preview details & customize the preview to open the editor.

**Comment by taipeicoder** [comment]
Also noting that with https://github.com/Automattic/wp-calypso/pull/85315, the demo site preview (and the live preview) will only be accessible via the Theme Details page once that project launches.

**Comment by miksansegundo** [comment]
+1 to put the Demo button back to the top.

I would also reconsider using "Preview" in "Preview & Customize" because in the Live preview, users preview their site pages and theme templates on the editor, not on the front, not with demo content, and that experience is very different.

When users explore the "Templates" section, the previews won't look great without the proper site demo content (categories, tags, posts with feature images...)

Have a look at these videos comparing the experience.

1. The experience when exploring the theme features on the Demo site:

https://github.com/Automattic/wp-calypso/assets/1881481/fb220739-6a57-4ed1-a0f8-0efb6ef4c70d

2. The experience "Previewing" the theme on the editor's preview,  in two parts:


https://github.com/Automattic/wp-calypso/assets/1881481/96f81435-6a6e-4dd0-9f60-3782f3e0b96d



https://github.com/Automattic/wp-calypso/assets/1881481/635adb6c-6d2f-45b4-84b6-bed5c3f5c880


cc: @matt-west

**Comment by okmttdhr** [comment]
>This is the discussion on the decision to put the the buttons this way: https://github.com/Automattic/dotcom-forge/issues/2391#issuecomment-1599401582.

Just to give some context 🙂, I'd like to share our previous discussion pbxlJb-3Uv-p2#comment-2868, pbxlJb-3Uv-p2#comment-2805 about displaying the 'Demo Site' button when a user hovers over the large preview. 

Also, I'm sharing the previous CfT feedback from those who **want** something like 'Demo Preview'!
- pekYwv-1WD-p2#comment-1516
- pekYwv-250-p2#comment-1592
- pekYwv-1WD-p2#comment-1604

**Comment by taipeicoder** [comment]
Closing this one as the current UI has significantly diverged from the time this issue was created. Presently, the Live Preview button is only available in the Theme Details page upon clicking the "Preview" button.

![Image](https://github.com/user-attachments/assets/c271e4e2-b77e-44a2-aea1-f44f781bb433)
